appium-dotnet-driver/Appium/Android/AndroidDriver.cs

```diff
@@ -255,5 +255,44 @@ public ConnectionType ConnectionType
                 }
             }
         }
+
+        /// <inheritdoc />
+        public void SetClipboard(ClipboardContentType contentType, string base64Content)
+        {
+            switch (contentType)
+            {
+                case ClipboardContentType.Image:
+                case ClipboardContentType.Url:
+                    throw new NotImplementedException($"Android only supports contentType: {nameof(ClipboardContentType.PlainText)}");
+                case ClipboardContentType.PlainText:
+                    AppiumCommandExecutionHelper.SetClipboard(this, contentType, base64Content);
+                    break;
+                default:
+                    AppiumCommandExecutionHelper.SetClipboard(this, contentType, base64Content);
+                    break;
+            }
+        }
+
+        /// <inheritdoc />
+        public void SetClipboardText(string textContent, string label) =>
+            AppiumCommandExecutionHelper.SetClipboardText(this, ClipboardContentType.PlainText, textContent, label);
+
+        /// <inheritdoc />
+        public string GetClipboard(ClipboardContentType contentType)
+        {
+            switch (contentType)
+            {
+                case ClipboardContentType.Image:
+                case ClipboardContentType.Url:
+                    throw new NotImplementedException($"Android only supports contentType: {nameof(ClipboardContentType.PlainText)}");
+                case ClipboardContentType.PlainText:
+                    return AppiumCommandExecutionHelper.GetClipboard(this, contentType);
```

Comment:
this clause might be merged with default to reduce duplication. Same for SetClipboard